piq/feature_extractors/clip.py

```diff
@@ -0,0 +1,617 @@
+import hashlib
+import os
+import warnings
+
+import torch
+import torch.nn.functional as F
+
+from torch import nn
+from typing import Tuple, Union, Optional
+from collections import OrderedDict
+from urllib.request import urlopen
+
+
+CLIP_MODEL_PATH = (
+    "https://openaipublic.azureedge.net/clip/models/"
+    "afeb0e10f9e5a86da6080e35cf09123aca3b358a0c3e3b6c78a7b63bc04b6762/RN50.pt"
+)
+
+
+def _download(url: str, root: str) -> str:
```

Comment:
That seems to be a functionality related not only to CLIP. Can we move this to /utils? 
We already load SVM weights from one of the releases in BRISQUE code, so that can be unified?

Reply:
I am not sure I want to refactor BRISQUE code as well. It is out of scope of this PR.